docs/sources/dashboards/_index.md

```diff
@@ -9,28 +9,53 @@ labels:
 title: Dashboards
 weight: 70
 description: Create and manage dashboards
+hero:
+  title: Dashboards
+  level: 1
+  width: 110
+  height: 110
+  description: >-
+    Dashboards allow you to query, visualize, alert on, and understand your data no matter where it’s stored. Translate and transform any of your data into flexible and versatile dashboards.
+cards:
+  title_class: pt-0 lh-1
+  items:
+    - title: Build dashboards
+      href: /docs/grafana/latest/dashboards/build-dashboards/
```

Comment:
I think we will want to use relative URLs so that these links don't take people from other versions into "latest" docs:

```suggestion
      href: ./build-dashboards/
```